LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2017 Franzé Jr.
```

Comment:
lets set teh copyright to Concur Messaging, Inc. for now, we can change it to Highlands later if they want